ReactWindows/ReactNative.Shared/Views/TextInput/ReactPasswordBoxShadowNode.cs

```diff
@@ -260,7 +271,7 @@ private static string GetDefaultPasswordChar()
             if (s_passwordChar == null)
             {
                 var passwordBox = new PasswordBox();
-                s_passwordChar = passwordBox.PasswordChar;
+                s_passwordChar = passwordBox.PasswordChar.ToString();
```

Comment:
Confirmed in ILSpy that the .ToString() call is "compiled away".  I don't think the C# compiler is in the business of removing .ToString() calls.  I did run a test to ensure the return value of .ToString() was a reference equal string (that it wasn't allocating a copy), so the only optimization is to avoid an unnecessary method call.

---
In reply to: [89868115](https://github.com/ReactWindows/react-native-windows/pull/900#discussion_r89868115) [](ancestors = 89868115)